connectors/utils.py

```diff
@@ -425,10 +425,10 @@ def _callback(self, task, result_callback=None):
                 exc_info=True,
             )
         if result_callback is not None:
-            result_callback(task.result())
+            result_callback(task)
         # global callback
         if self.results_callback is not None:
-            self.results_callback(task.result())
+            self.results_callback(task)
```

Comment:
This was broken. `task.result()` throws if the task has an exception. And since this exception doesn't have an error handling, any task errors just get silently swallowed into the void.

Reply:
I checked and did not find any usage of result callbacks in code, might be worth to just delete it?

Reply:
Ok, I've removed the _result_callback stuff from ConcurrentTasks.

It was used heavily in the tests, so you'll still see callbacks there, but I figure that's preferable to totally rewriting all those tests.